x-pack/legacy/plugins/apm/server/routes/typings.ts

```diff
@@ -49,13 +49,7 @@ export interface Route<
   }) => Promise<TReturn>;
 }
 
-export type APMLegacyServer = Pick<Server, 'usage' | 'savedObjects' | 'log'> & {
-  usage: {
-    collectorSet: {
-      makeUsageCollector: (options: unknown) => unknown;
-      register: (options: unknown) => unknown;
-    };
-  };
+export type APMLegacyServer = Pick<Server, 'savedObjects' | 'log'> & {
```

Comment:
Yay, no more custom types. I think actually you can remove all of it so:

```ts
export type APMLegacyServer = Server;
```

Reply:
FWIW, this was per suggestion of @rudolf, to be very explicit about whatever functionality we are still using from `Server`, so we have better control over what we allow `Server` to be used for. I personally still think that that is a good idea